cc-cloudfuse

```diff
@@ -7,6 +7,13 @@ fi
 shift
 ARGS=$@
 
+# check auth url
+if ! [ -z ${OS_AUTH_URL:+x} ]; then
+    if ! [[ $OS_AUTH_URL == *?v3 ]]; then
```

Comment:
I'm not familiar with the `?` syntax, where did you find that?

FYI you can use `!=` here as well, they should be equivalent in this case.

Reply:
https://www.guru99.com/linux-regular-expressions.html
need to contain something before v3.